### PR TITLE
Add tests #15

### PR DESCRIPTION
**Pull request by TinaHeiligers**
Sets up initial test examples with testing environment working.
Adds test section to README.

### CODE REVIEW

**Comment by TinaHeiligers** [comment]
@krantzinator @vinnieking06 Again, just FYI.

File: test/routes/home.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    };
+    const result = await server.inject(options);
+    expect(result).to.exist();
+    expect(result.statusCode).to.equal(200);
+  });
+});
+describe("routes -> '/restricted'", () => {
+  it('permits authorized users to access the route', async () => {
+    const server = await Server.deployment();
+    const testUserName = mockedUsers[0].username; // do not do this at home folks! Use factory created instances of your user model.
```
Comment (krantzinator) [inline test/routes/home.js]: are you going to add a factory later?

Comment (vinnieking06) [reply test/routes/home.js]: 😂

**Review by vinnieking06 — APPROVED** [review]
Nice work @TinaHeiligers I will certainly use this if I need to create an auth system.

**Pull request merged**